### ISSUE TITLE
ISPC support?

### ISSUE THREAD

**Issue by Zerophase** [ccache/ccache.github.io#5]
Unreal uses a lot of ISPC compile calls, on the main project. Think that's largely why I get very few cache hits when doing a full engine recompile. Ccache greatly improves compile speed on any target, without ISPC, but probably slows compilation down on the main project. Any plans on adding further support for ISPC?

**Comment by jrosdahl** [comment]
No, there are no plans to add support for ISPC. The development of Ccache is mainly driven by the interest of individuals in the community, so contributions of such support would be welcome unless they are too intrusive or complex.

Meta note: This issue tracker is about the content of the Ccache _web site_, not for questions about Ccache itself. I suggest using the [Ccache discussions](https://github.com/ccache/ccache/discussions) forum for questions like this.
